linux/install_aws_cli.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+# Install AWS CLI v2
+echo "Install AWS CLI v2.."
+mkdir -p /tmp/awscliv2
+curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "/tmp/awscliv2/awscliv2.zip"
+pushd /tmp/awscliv2 && unzip ./awscliv2.zip && sudo ./aws/install && popd
+echo "Done"
+
+# Install SSM session manager plugin
+echo "Install the AWS CLI Session Manager Plugin.."
+mkdir -p /tmp/ssm
+curl "https://s3.amazonaws.com/session-manager-downloads/plugin/latest/mac/sessionmanager-bundle.zip" -o "/tmp/ssm/sessionmanager-bundle.zip"
+pushd /tmp/ssm && unzip ./sessionmanager-bundle.zip && sudo ./sessionmanager-bundle/install -i /usr/local/sessionmanagerplugin -b /usr/local/bin/session-manager-plugin && popd
```

Comment:
At line 6 you're installing the linux version of the cli, but at line 13 you're installing the mac version of the session manger plugin. Since AWS provides different installers for linux vs. mac, it seems that either you need to create a separate file for mac. Or make the script smart enough to detect OS (such as with `uname -s`).

Reply:
are you correct.  hmm, i guess would be easiest just to create a separate set of files for linux and mac.  another option would be to convert to a python script however that would require the user to know about python.

Reply:
hmm, wondering if we even need to provide this script at all?  Maybe we can just assume users can handle installing the AWS CLI and session manager plugin by following instructions from AWS docs?  what do you think @tthyer?

Reply:
Honestly, I feel the feedback we were getting is that they would like a convenience script for this. Maybe ask Kevin too.